cmd/sink/main.go

```diff
@@ -119,6 +119,17 @@ func (sink *Sink) Receive(ctx context.Context, event cloudevents.Event) {
 			return
 		}
 		sink.Logger.Println("successfully deleted kubernetes object:", k8sObj.GetUID())
+		deleteTs := metav1.Now()
+		k8sObj.SetDeletionTimestamp(&deleteTs)
+		sink.Logger.Printf("updating database entry for kubernetes object %s after successful deletion\n", k8sObj.GetUID())
+		updateCtx, updateCancel := context.WithTimeout(context.Background(), time.Second*5)
+		err = sink.Db.WriteResource(updateCtx, k8sObj, event.Data())
+		defer updateCancel()
+		if err != nil {
+			sink.Logger.Printf("failed to update database entry for kubernetes object %s after successful deletion\n", k8sObj.GetUID())
```

Comment:
This comment applies to this log message and also one earlier in the file that is not part of this change.

It's not really a successful deletion.  The message implies the deletion is done, but it's not.  It's really a successful submit of the deletion request to kubernetes.

I think this message could actually say something shorter, like "failed to update `ts_deleted`".  And earlier "updating `ts_deletion` for resource uuid  <primary-key>"

Reply:
fixed the log messages